manifests/install.pp

```diff
@@ -9,11 +9,19 @@
   $carepo         = $fetchcrl::carepo,
   $manage_carepo  = $fetchcrl::manage_carepo,
   $capkgs_version = $fetchcrl::capkgs_version,
-  $pkg_version    = $fetchcrl::pkg_version
+  $pkg_version    = $fetchcrl::pkg_version,
+  $inet6glue      = $fetchcrl::inet6glue
 ) {
   assert_private()
 
   # The fetch-crl package.
+  if $inet6glue {
+    package { 'perl-Net-INET6Glue':
```

Comment:
Once rebased this  will need to install `libnet-inet6glue-perl` for `$facts['os']['family'] == 'Debian' case.

Reply:
Updated accordingly.